src/com/ableton/Pyenv.groovy

```diff
@@ -23,7 +23,7 @@ class Pyenv implements Serializable {
 
   Pyenv(Object script, String pyenvRoot) {
     this.script = script
-    this.pyenvRoot = pyenvRoot
+    this.pyenvRoot = pyenvRoot ? VirtualEnv.posixPath(pyenvRoot) : null
```

Comment:
Fixup for f93cbac65f60334e86e93d5a959ef386103fbd09 (whose commit message should also be reworded)
```suggestion
    this.pyenvRoot =
      script.env.OS == 'Windows_NT' ? pyenvRoot.replace('\\', '/') : pyenvRoot
```
It might actually make sense to simplify squash that change into "Support pyenv on Windows".